core/src/test/java/hudson/PluginWrapperTest.java

```diff
@@ -95,6 +100,35 @@ public void dependencyFailedToLoad() {
         assertContains(ex, "Failed to load: dependee (42)", "Failed to load: dependency (5)");
     }
 
+    @Issue("JENKINS-66563")
+    @Test
+    public void insertJarsIntoClassPath() throws Exception {
+        try (AntClassLoader cl = new AntClassLoader()) {
+            assertInjectingJarsWorks(cl);
+        }
+    }
+
+        @Issue("JENKINS-66563")
```

Comment:
indenting looks off for this test

Reply:
fixed in 543ba52325f20cf55fd9fb8906909cc48b2e7fd6